en-US/snippets/2017/oct2017.lang

```diff
@@ -0,0 +1,99 @@
+# Link: https://www.mozilla.org/firefox/quantum/
+;See the future of Firefox, no DeLorean required. <a>Check out Quantum.</a>
+See the future of Firefox, no DeLorean required. <a>Check out Quantum.</a>
+
+
+# Snippet for Firefox quantum
+# “aka” is the acronym for “also known as”. Feel free to substitute it with the equivalent in your language.
+# Link: https://www.mozilla.org/firefox/quantum/
+;Don’t miss out — Firefox Quantum (aka the best Firefox ever) launches soon. <a>Get notified when it becomes available.</a>
+Don’t miss out — Firefox Quantum (aka the best Firefox ever) launches soon. <a>Get notified when it becomes available.</a>
+
+
+# Snippet for Firefox features
+# The repeat use of the word "fast" is to convey to users that Firefox is the speedest it's ever been.
+;The new Firefox can be summed up in two words: faster and faster. OK, maybe that’s the same word twice.<a>Learn More.</a>
+The new Firefox can be summed up in two words: faster and faster. OK, maybe that’s the same word twice.<a>Learn More.</a>
+
+
+# Snippet for Firefox features
+# The repeat use of the word "fast" is to convey to users that Firefox is the speedest it's ever been.
+;The new Firefox is all about fast. Faster startups. Faster tab switching. Faster scrolling. Faster at letting you get more done. <a>Learn more.</a>
+The new Firefox is all about fast. Faster startups. Faster tab switching. Faster scrolling. Faster at letting you get more done. <a>Learn more.</a>
+
+
+# Snippet for Firefox features
+# Tab hoarding is the practice of having a lot of browser tabs open.
+;<a>Firefox is refreshingly fast,</a> with quicker startup times, fewer crashes and better at making tab hoarders happy (you know who you are.)
+<a>Firefox is refreshingly fast,</a> with quicker startup times, fewer crashes and better at making tab hoarders happy (you know who you are.)
+
+
+# Snippet for Firefox
+;Firefox is the only browser that’s made by a non-profit, so it doesn’t sell access to your every click. #Nowyouknow
+Firefox is the only browser that’s made by a non-profit, so it doesn’t sell access to your every click. #Nowyouknow
+
+
+# Snippet for Firefox
+;Mozilla is a champion of the open web. Firefox is made by Mozilla. So every time you use Firefox, (like right now) you’re helping to make the web better for everyone.
+Mozilla is a champion of the open web. Firefox is made by Mozilla. So every time you use Firefox, (like right now) you’re helping to make the web better for everyone.
+
+
+# Snippet for form fill feature
+# Link: https://support.mozilla.org/kb/control-whether-firefox-automatically-fills-forms
+;Firefox can <a>fill out forms for you,</a> so you have more time to do important stuff online, like look at memes.
+Firefox can <a>fill out forms for you,</a> so you have more time to do important stuff online, like look at memes.
```

Comment:
No sense in promoting a feature, with a localized snippet, that is only available to en-US builds in US territory.

Reply:
i see German with a screenshot, but not sure about other languages on this one. the article is localized in other languages.

Reply:
It's not the article. The feature is only enabled for en-US in US.